src/array-sort.ts

```diff
@@ -0,0 +1,33 @@
+export function arraySort<T>(sortByProperty: string, isDescending = false) {
+  const cachedValueKey = Symbol();
```

Comment:
Sorry, I didn't saw this PR.

Reply:
No worries at, I will address comments asap :)